### PR TITLE
add unsafe flag for subql/node

### PR DESCRIPTION
**Pull request by lachlan1m**
I have tested to make sure this flag works and with it you can for example make http requests. Although It was not so straight forward to make http requests in subql test project.

### CODE REVIEW

**Comment by jamesbayly** [comment]
Can you also update the docs for this change?

File: packages/node/src/indexer/sandbox.service.ts [hunk 1]
```diff
@@ -28,7 +31,9 @@ const DEFAULT_OPTION: NodeVMOptions = {
   wasm: false,
```
Comment (ianhe8x) [inline packages/node/src/indexer/sandbox.service.ts:31]:
```suggestion
  wasm: argv.unsafe,
```

File: packages/node/src/yargs.ts [hunk 2]
```diff
@@ -34,6 +34,11 @@ export function getYargsOption() {
       demandOption: false,
       describe: 'Force clean the database, dropping project schemas and tables',
     },
+    unsafe: {
+      type: 'boolean',
+      demandOption: false,
+      describe: 'Allows usage of any built-in module for sandbox service',
```
Comment (ianhe8x) [inline packages/node/src/yargs.ts:40]:
```suggestion
      describe: 'Allows usage of any built-in module within the sandbox',
```

File: packages/node/src/main.ts [hunk 3]
```diff
@@ -21,6 +21,13 @@ async function bootstrap() {
     const indexerManager = app.get(IndexerManager);
     await indexerManager.start();
     await app.listen(port);
+
+    if (argv('unsafe')) {
```
Comment (ianhe8x) [inline packages/node/src/main.ts:25]: move it before `const app = await NestFactory.create`

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**